go/cmd/dolt/commands/tblcmds/import.go

```diff
@@ -68,6 +68,20 @@ const (
 	disableFkChecks   = "disable-fk-checks"
 )
 
+var jsonInputFileHelp = "A mapping file is json in the format:" + `
```

Comment:
Close, but this text should be something like "when using a JSON input file, the expected format is:"